osbenchmark/benchmark.py

```diff
@@ -566,6 +566,21 @@ def add_workload_source(subparser):
              f"(default: {metrics.GlobalStatsCalculator.DEFAULT_THROUGHPUT_PERCENTILES}).",
         default=metrics.GlobalStatsCalculator.DEFAULT_THROUGHPUT_PERCENTILES
     )
+    test_execution_parser.add_argument(
+        "--randomization-enabled",
+        help="Runs the given workload with query randomization enabled (default: false).",
+        default=False,
+        action="store_true")
+    test_execution_parser.add_argument(
+        "--randomization-rf",
```

Comment:
We should aim to use full-length arguments as best practice, such as `--randomization-repeat-frequency`, in addition to the short-hand format, like `--randomization-rf` or `-rrf`. That way users don't have to look up what the meaning of the flags as much.

Reply:
@peteralfonsi Integration tests are failing due to extra parameter added has `--` and is not considered a "short-form" flag. Think you'll need to change this to "-rf" or some iteration with a single hyphen.

Reply:
Ahh - thanks, that probably would have taken me a while to figure out!

Reply:
After second glance, `--randomization-repeat-frequency` is a bit lengthy and can be cumbersome for users to type out every time. Short-form flags should be used for more commonly used flags like `--target-hosts` or `--workload`. `--randomization-repeat-frequency` is a specific use-case and I think it will be fine to keep it as `--randomization-rf`.

For this option, we can either do what you did before and keep `--randomization-rf` or we can have the full-length version `--randomization-repeat-frequency` and a short-form version `-rf`. I'll leave it up to your preference!